README.md

```diff
@@ -26,6 +26,7 @@ metadata:
   name: provider-kubernetes
 spec:
   package: xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.13.0
+  # package: xpkg.upbound.io/upbound/provider-kubernetes:v0.16.0 for v0.16.0 or higher
```

Comment:
i'd say we can just make this update completely, as opposed to adding a new commented line. that's the new location and we'd like folks to be installing the latest, so let's just update these references to the most newest location/version.

Reply:
it installed OK/healthy for me 😇 
```
❯ k get pkg
NAME                                             INSTALLED   HEALTHY   PACKAGE                                               AGE
provider.pkg.crossplane.io/provider-kubernetes   True        True      xpkg.upbound.io/upbound/provider-kubernetes:v0.16.0   48m
```

Reply:
there's also a reference using `crossplane xpkg install` a few lines above, you can make that update too as part of this PR:
```
crossplane xpkg install provider xpkg.upbound.io/upbound/provider-kubernetes:v0.16.0
```